pkg/sql/pg_catalog.go

```diff
@@ -1153,6 +1153,31 @@ https://www.postgresql.org/docs/9.5/catalog-pg-depend.html`,
 					depTypeAuto,          // deptype
 				)
 			}
+
+			reportViewDependency := func(dep *descpb.TableDescriptor_Reference) error {
```

Comment:
how does this code handle views that depend on other views? can you add a test for that?

Reply:
The loop already pick views and descriptor already have dependencies at other views, but this is a nice catch, I tried it and it works, I am going to add a test case for this as well.

Reply:
Done